p2p/switch_test.go

```diff
@@ -311,8 +314,11 @@ func BenchmarkSwitches(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		chID := byte(i % 4)
 		successChan := s1.Broadcast(chID, "test data")
-		for s := range successChan {
-			if s {
+		for res := range successChan {
+			if !s1.peers.Has(res.PeerKey) {
```

Comment:
is this the only place we're using `res.PeerKey`? so it's only for tests? what is we testing exactly?